exercises/diffie-hellman/.meta/HINTS.md

```diff
@@ -0,0 +1,18 @@
+# Hints
```

Comment:
Could you please rename the file to `hints.md` (lowercase) for consistency

Reply:
Done! I'd done it but apparently there was some error in renaming